src/routes/profile/[id=profile_id]/+layout.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  import pencilIcon from '$lib/icons/pencil.svg'
+  import Header from './Header.svelte'
+  import PersonalInformation from './PersonalInformation.svelte'
+  import PersonalLinks from './PersonalLinks.svelte'
+
+  export let data
+
+  let disabledExport = false
+
+  async function exportCV() {
+    try {
+      disabledExport = true
+      const res = await fetch(`/api/profile/generate-cv`, {
+        method: 'POST',
+        body: JSON.stringify({ id: data.userId })
+      })
+
+      const pdfBlob = await res.blob()
+
+      downloadPDF(pdfBlob)
+    } catch (e) {
+      alert(e)
+    } finally {
+      disabledExport = false
+    }
+  }
+
+  function downloadPDF(pdfBlob: Blob) {
+    const url = window.URL.createObjectURL(pdfBlob)
+
+    const link = document.createElement('a')
+    link.href = url
+    link.download = 'CV.pdf'
```

Comment:
```suggestion
    link.download = `${data.name.split(' ').join('-').toLowerCase()}.pdf`
```
Creo que se vería mejor así el nombre del CV (Nombre de la persona en kebab case). De igual manera, esto cambiará luego otra vez